ui/app/AppLayouts/Chat/views/ChatView.qml

```diff
@@ -89,9 +89,6 @@ StatusSectionLayout {
         onOpenStickerPackPopup: {
             Global.openPopup(statusStickerPackClickPopup, {packId: stickerPackId, store: root.stickersPopup.store} )
         }
-        onOpenAppSearch: {
-            root.openAppSearch();
```

Comment:
Not needed any longer?

Reply:
Well it just wasn't ever called from `ChatContenView` 🙂